lib/cosmos/tools/script_runner/script_runner_frame.rb

```diff
@@ -151,6 +151,16 @@ def initialize(parent, default_tab_text = 'Untitled')
       @realtime_button_bar.stop_callback  = method(:handle_stop_button)
       @layout.addWidget(@realtime_button_bar)
 
+      # Create shortcuts to activate the RealtimeButtonBar actions
+      step = Qt::Shortcut.new(Qt::KeySequence.new(Qt::Key_F10), self)
+      self.connect(step, SIGNAL('activated()')) { handle_step_button() }
+      start_go = Qt::Shortcut.new(Qt::KeySequence.new(Qt::Key_F5), self)
+      self.connect(start_go, SIGNAL('activated()')) { handle_start_go_button() }
+      pause_retry = Qt::Shortcut.new(Qt::KeySequence.new(Qt::Key_F6), self)
+      self.connect(pause_retry, SIGNAL('activated()')) { handle_pause_retry_button() }
+      stop = Qt::Shortcut.new(Qt::KeySequence.new(Qt::Key_F7), self)
+      self.connect(stop, SIGNAL('activated()')) { handle_stop_button() }
```

Comment:
F10 seems common for Step as does F5 for start. Not sure about F6 for pause retry and F7 for stop but they were next in line. Ideally we would allow users to map whatever key they want but that seemed like a lot of extra complexity I didn't want to tackle now.